pyscf/gto/basis/soecp/ECPDS78MDFSO.dat

```diff
@@ -0,0 +1,127 @@
+ECP
+# CHECKME 78? 60?
+# Q=22., MEFIT, MCDHF+Breit+QED, Ref 29.
+# [29] B. Metz, H. Stoll, M. Dolg, J. Chem. Phys. 113, 2563 (2000).
+Pb nelec 60
```

Comment:
This should probably be 78

Reply:
Yes, you are right. The dirac's ecp library has this ambiguity, maybe I should tell them.